scripts/elevators.zs

```diff
@@ -2,23 +2,40 @@ import crafttweaker.liquid.ILiquidStack;
 import crafttweaker.item.IItemStack;
 
 // remove
-recipes.removeByMod("elevatorid");
+val elevator_recipes = [
+    "elevatorid:elevator_white",
+    "elevatorid:elevator_orange",
+    "elevatorid:elevator_magenta",
+    "elevatorid:elevator_light_blue",
+    "elevatorid:elevator_yellow",
+    "elevatorid:elevator_lime",
+    "elevatorid:elevator_pink",
+    "elevatorid:elevator_gray",
+    "elevatorid:elevator_silver",
+    "elevatorid:elevator_cyan",
+    "elevatorid:elevator_purple",
+    "elevatorid:elevator_blue",
+    "elevatorid:elevator_brown",
+    "elevatorid:elevator_green",
+    "elevatorid:elevator_red",
+    "elevatorid:elevator_black"
+] as string[];
+
+for i, name in elevator_recipes {
+    recipes.removeByRecipeName(elevator_recipes[i]);
+}
 
 // Gray Elevator (Base)
-assembler.recipeBuilder()
-    .inputs(<ore:frameGtInvar> * 2)
-    .inputs(<gregtech:turbine_casing:1>)
-    .inputs(<metaitem:electric.motor.lv> * 2)
-    .inputs(<metaitem:emitter.lv>)
-    .outputs(<elevatorid:elevator_gray> * 4)
-    .duration(200)
-    .EUt(15)
-    .buildAndRegister();
+recipes.addShaped("elevator_gray", <elevatorid:elevator_gray> * 4, [
+    [<ore:plateSteel>, <ore:plateSteel>, <ore:plateSteel>],
+    [<ore:gearSmallSteel>, <ore:frameGtSteel>, <ore:gearSmallSteel>],
+    [<ore:plateSteel>, <metaitem:emitter.lv>, <ore:plateSteel>]
```

Comment:
Elevators absolutely should not use emitters. The cost is far too much. Please replace this with a steel plate.

Reply:
A motor is not any better. No one is going to want to make these until HV tier, and even then, it doesn't feel great. If you really insist on making these only reasonable at HV, change the recipe to an enderpearl plate and gate it at HV instead. 

This should really just be another steel plate.